runtime-services-modeling/decisions-with-dmn.adoc

```diff
@@ -1520,11 +1524,6 @@ Click the check mark to the right of each data type that you create to save your
 
 .Adding nested data types with constraints
 image::dmn-manage-data-types-structured2.png[]
-
-To change the order or nesting of data types, you can click the left end of the data type row and drag the row as needed:
-
-.Dragging data types to change data type order or nesting
-image::dmn-manage-data-types-structured2-drag.png[]
```

Comment:
- picture was outdated due to new design of data types editor
- it is nice to have feature, reordering data type fields, order of fields doesn't affect the model execution
- removing to reduce length of the docs and reduce screenshots maintenance costs